src/main/java/edu/kis/vh/nursery/stack/IntArrayStack.java

```diff
@@ -0,0 +1,42 @@
+package edu.kis.vh.nursery.stack;
+
+public class IntArrayStack implements Stacker {
+
+    private int[] NUMBERS = new int[MAX_STACK_SIZE];
+    private int total = INIT_EMPTY_STACK;
```

Comment:
Bugs resulting from extracting constants still exist. This value should never change to 0.